Matveibos/hometask-4/my_hash.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+# this class is Hash
+class MyHash
+  include Enumerable
+
+  attr_reader :pair_count, :arr
+
+  LOAD_FACTOR = 0.7
+  INITIAL_SIZE = 10
+  FOUR = 4
```

Comment:
I know that this is four. Maybe we should name this constant to say more about purpose of number 4 in our code?
Also, add empty line after constants